config.yaml

```
@@ -0,0 +1,145 @@
+baseURL: https://scipy.scientific-python.org/
+languageCode: en-us
+theme: scientific-python-hugo-theme
+disableKinds: ["taxonomyTerm"]
+googleAnalytics: UA-162151853-1
+DefaultContentLanguage: en
+markup:
+  goldmark:
+    renderer:
+      unsafe: true
+
+# Uncomment this to disable the translations for the specified languages.
+# This may be useful to be able to merge partial translations without deploying
+# them to numpy.org immediately.
+disableLanguages: ["pt", "ja"]
+
+params:
+  images:
+  - /images/logo.svg
+  navColor: blue
+  font:
+    name: "Lato"
+    sizes: [400,900]
+
+languages:
+  en:
+    title: SciPy
+    weight: 1
+    contentDir: content/en
+
+    languageName: English
+    params:
+      description: Why SciPy? Fundamental algorithms. Broadly applicable. Foundational. Interoperable. Performant. Open source.
+      navbarlogo:
+        image: logo.svg
+        altText: A blue circle with a snake in the shape of the letter 'S'.
+        link: /
+      hero:
+        # Main hero title
+        title: SciPy
+        # Hero subtitle (optional)
+        subtitle: Fundamental algorithms for scientific computing in Python
+        # Button text
+        buttontext: Get started
+        # Where the main hero button links to
+        buttonlink: "/install"
+        # Hero image (from static/images/___)
+        image: logo.svg
+      # Customizable navbar. For a dropdown, add a "sublinks" list.
+      news:
+        title: SciPy 1.7.1 released
+        content: 2021-08-01
+        url: /news
+
+      #shell:
+      #  title: placeholder
+
+      #  promptlabel: interactive shell prompt
+
+      #  button:
+      #  - label: Enables the interactive tutorial shell
+      #    text: Enable
+
+      #shellcontent:
+      #  intro:
+      #  - title: Try SciPy
+      #    text: Enable the interactive shell
+
+      #  loading:
+      #  - title: While we wait...
+      #    text: Launching container on mybinder.org...
+
+      #  docslink: Don't forget to check out the <a href="https://docs.scipy.org/doc/scipy/reference/" target="_blank">docs</a>.
+
+      keyfeatures:
+        features:
+        - title: Fundamental algorithms
+          text: SciPy provides algorithms for optimization, integration, interpolation, eigenvalue problems, algebraic equations, differential equations and many other classes of problems.
```

I would add statistics since it's a big module and should help SEO??
```suggestion
          text: SciPy provides algorithms for optimization, integration, interpolation, eigenvalue problems, algebraic equations, differential equations, statistics and many other classes of problems.
```